llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -108,7 +108,7 @@ using namespace slpvectorizer;
 
 STATISTIC(NumVectorInstructions, "Number of vector instructions generated");
 
-cl::opt<bool> RunSLPVectorization("vectorize-slp", cl::init(true), cl::Hidden,
+cl::opt<bool> RunSLPVectorization("vectorize-slp", cl::init(false), cl::Hidden,
```

Comment:
Can you add a comment here mentioning what you wrote in the PR abstract pls.

Reply:
done.